src/com/google/foam/demos/olympics/Controller.js

```diff
@@ -81,34 +88,43 @@ foam.CLASS({
           'color',
           'year',
           'city',
-          'discipline',
+          'sport',
           'event',
           'country',
           'gender'
         ];
       }
     },
-    'scroller_'
+    'scroller_',
+    'container_',
+    'scrollExtent',
+    'scrollHeight'
   ],
 
   methods: [
     function tableE(parent) {
       var self = this;
       var canvas = this.Canvas.create(null, parent);
+      canvas.attrs({ height: this.scrollHeight$ });
 
       var scroller = this.ScrollCView.create({
+        borderColor: '#e0e0e0',
+        handleColor: '#3f51b5',
         size: 0,
-        //size: 1000,
-        extent: 10
+        extent$: this.scrollExtent$,
+        height$: this.scrollHeight$
       }, canvas);
       canvas.cview = scroller;
       this.scroller_ = scroller;
 
+      this.window.addEventListener('resize', this.onResize);
+
       parent.on('wheel', function(e) {
         scroller.value += e.deltaY > 0 ? 3 : -3;
```

Comment:
I think the returned value should be a function of deltaY.  If the browser batches up a bunch of wheel events and delivers one with a deltaY that is large, I'd expect my table to scroll an equal amount because that more cloesly matches the distance I actually turned the wheel.